build.gradle

```diff
@@ -272,8 +272,12 @@ task startStoragesInDocker() {
     outputs.upToDateWhen { false }
 }
 
+task generateScmSourceJson(type: Exec) {
+    commandLine "scm-source", "-f", "scm-source.json"
```

Comment:
I think that we need to build this file each time we create docker image:
 - scm-source is a public tool, and a public way to describe version (no need to move to private repo)
 - scm-source is a way to describe what's in box (so it's better to include version every time we build image, as it's a part of docker image).